### PR TITLE
Semihalf updates

### PR DESCRIPTION
**Pull request by semihalf-wojtas-marcin**
Hi,

Please check the 2020Q1 updates from the Semihalf team.

Best regards,
Marcin Wojtas mw@

### CODE REVIEW

File: 2020q1/ena.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+Contact: Marcin Wojtas <mw@semihalf.com>  
+
+ENA (Elastic Network Adapter) is the smart NIC available in the
+virtualized environment of Amazon Web Services (AWS). The ENA
+driver supports multiple transmit and receive queues and can handle
+up to 100 Gb/s of network traffic, depending on the instance type
+on which it is used.
+
+Completed since the last update:
+  * Upstream of the driver v2.1.1 version, introducing:
```
Comment (debdrup) [inline 2020q1/ena.md:16]: s/driver v2.1.1 version/driver to v2.1.1/

File: 2020q1/ls1046.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+The Semihalf team initiated working on FreeBSD support for the
+[NXP LS1046A SoC](https://www.nxp.com/products/processors-and-microcontrollers/arm-based-processors-and-mcus/qoriq-layerscape-arm-processors/qoriq-layerscape-1046a-and-1026a-multicore-communications-processors:LS1046A)
+
+LS1046A are quad-core 64-bit ARMv8 Cortex-A72 processors with
+integrated packet processing acceleration and high speed peripherals
+including 10 Gb Ethernet, PCIe 3.0, SATA 3.0 and USB 3.0 for a wide
+range of networking, storage, security and industrial applications.
+
+Completed since the last update:
+  * Clean-up and rebase support on top of the FreeBSD-HEAD. Prepare features
```
Comment (debdrup) [inline 2020q1/ls1046.md:16]: s/of the/of/

Comment (semihalf-wojtas-marcin) [reply 2020q1/ls1046.md]: Done

File: 2020q1/ena.md (hunk 1 above)
Comment (semihalf-wojtas-marcin) [reply 2020q1/ena.md]: Done

**Pull request merged**